application/testing/CMakeLists.txt

```diff
@@ -1104,8 +1104,8 @@ endif()
 f3d_test(NAME TestRenderingBackendInvalid DATA cow.vtp RENDERING_BACKEND invalid ARGS --verbose REGEXP "rendering-backend value is invalid, falling back to" NO_BASELINE)
 
 # Test that f3d can try to read a system config file
-add_test(NAME f3d::TestNoDryRun COMMAND $<TARGET_FILE:f3d> --no-render)
-set_tests_properties(f3d::TestNoDryRun PROPERTIES TIMEOUT 4)
+add_test(NAME f3d::TestNoNoConfig COMMAND $<TARGET_FILE:f3d> --no-render)
+set_tests_properties(f3d::TestNoNoConfig PROPERTIES TIMEOUT 4)
 
```

Comment:
What about adding the `TestNoConfigWithConfig` we discussed earlier ?

Reply:
Hi,

I procrastinated learning ctest.
I wanted to run the test locally before pushing